.github/workflows/run-checks.yml

```diff
@@ -18,12 +18,16 @@ jobs:
     strategy:
       matrix:
         include:
-          - environment: "py36"
-            python: "3.6"
-          - environment: "py37"
-            python: "3.7"
           - environment: "py38"
             python: "3.8"
+          - environment: "py39"
```

Comment:
No need to change, but realized that `matrix.environment` is not used at all, so this matrix could be simplified:

```yaml
matrix:
  python: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```